### PR TITLE
Add a few more en_GB -> en_US suggestions

### PR DESCRIPTION
**Pull request by lurch**

### CODE REVIEW

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 1]
```diff
@@ -30,6 +31,7 @@ criticise->criticize
 criticised->criticized
 criticises->criticizes
 crystallisation->crystallization
+crystallise->crystallize
```
Comment (sebweb3r) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:34]: crystallized, crystallizes, crystallization?

Comment (lurch) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:34]: crystallization is already in there :stuck_out_tongue_winking_eye:  But I'll add the other two :+1:

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 2]
```diff
@@ -78,58 +134,112 @@ mitre->miter
...
 optimisation->optimization
 optimisations->optimizations
+optimise->optimize
 optimised->optimized
 optimiser->optimizer
+optimises->optimizes
+optimising->optimizing
 organisation->organization
 organisations->organizations
 organise->organize
```
Comment (peternewman) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:155]: Organiser

**Review by peternewman — CHANGES_REQUESTED** [review]
A few more suggestions

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 3]
```diff
@@ -10,35 +15,74 @@ artefact->artifact
...
 criticised->criticized
 criticises->criticizes
+criticising->criticizing
 crystallisation->crystallization
+crystallise->crystallize
+crystallised->crystallized
+crystallises->crystallizes
+crystallising->crystallizing
+customisation->customization
+customise->customize
```
[the remark this answers is not in the document]
Comment (lurch) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:60]: customisation already appears to be there? :laughing:

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 4]
```diff
@@ -10,35 +15,74 @@ artefact->artifact
...
+emphasises->emphasizes
+emphasising->emphasizing
 endeavour->endeavor
+endeavours->endeavors
+favour->favor
+favourite->favorite
+favourites->favorites
+favouritism->favoritism
+favours->favors
 finalise->finalize
```
[the remark this answers is not in the document]
Comment (lurch) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:82]: Dunno, that feels like a made-up word? :shrug:

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 5]
```diff
@@ -78,58 +136,116 @@ mitre->miter
...
 reinitialise->reinitialize
 reinitialised->reinitialized
 reorganisation->reorganization
+reorganise->reorganize
 reorganised->reorganized
+reorganises->reorganizes
+reorganising->reorganizing
 sanitise->sanitize
 sanitised->sanitized
+sanitiser->sanitizer
```
Comment (lurch) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:199]: Rather appropriate at the moment! :rofl: :microbe:

**Comment by lurch** [comment]
I wonder if it'd be useful for the check-sorting step to point out _all_ the mis-sorted words? Rather than just printing out the first error and then stopping, and not printing the second sorting error until after you've pushed a fix for the first sorting error.

**Comment by lurch** [comment]
According to the Travis check, neither customiser nor digitiser are in the aspell dictionary? :confused:

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (sebweb3r) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:84]: https://dictionary.cambridge.org/de/worterbuch/englisch/finalization?q=finalisation

[the remark this answers is not in the document]
Comment (lurch) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:85]: Okay, added

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt (hunk 2 above)
Comment (lurch) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:158]: Also added organisers, as in "The organisers of the event".

**Comment by peternewman** [comment]
> I wonder if it'd be useful for the check-sorting step to point out _all_ the mis-sorted words? Rather than just printing out the first error and then stopping, and not printing the second sorting error until after you've pushed a fix for the first sorting error.

It probably would, but I'm not sure that's easily possible given we're just abusing sort, unless you wrote something more complicated around it with a while loop.

If you're doing it on a computer not via the website, then `make sort-dictionaries` is your friend, to correctly sort it.

**Comment by peternewman** [comment]
> According to the Travis check, neither customiser nor digitiser are in the aspell dictionary? confused

The former should be fixed by #1485, the latter shows up the flaw with that and our dictionary work, as it's in the en_GB-large but not en_US-large dictionaries. Probably best dropping them for now and opening a new PR with just those two in.

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 6]
```diff
@@ -10,35 +15,74 @@ artefact->artifact
...
+crystallising->crystallizing
+customisation->customization
+customise->customize
 customised->customized
+customises->customizes
 customising->customizing
 defence->defense
 dialogue->dialog
+digitisation->digitization
 digitise->digitize
```
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:67]: Pulled out for now due to aspell issues.

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary_en-GB_to_en-US.txt:60]: Pulled out for now due to aspell issues.

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 7]
```diff
@@ -10,35 +15,75 @@ artefact->artifact
 artefacts->artifacts
 authorisation->authorization
 authorise->authorize
+authorised->authorized
+authorises->authorizes
```
Comment (peternewman) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:19]: Do you want to add -er

**Review by peternewman — CHANGES_REQUESTED** [review]
One more comment.

Can you fix the conflicts too please.

**Comment by lurch** [comment]
Done & done

**Comment by lurch** [comment]
LOL, Travis is complaining about `error 'authoriser' should be in aspell for dictionary /home/travis/build/codespell-project/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt`

Should I take it out again?

**Comment by sebweb3r** [comment]
Yes, because otherwise all the test will fail forever!

**Comment by sebweb3r** [comment]
I cherry-picked it in 1626, so we can sort it sometimes in the far future :smile:

**Review by peternewman — APPROVED** [review]
LGTM

**Comment by peternewman** [comment]
@sebweb3r are you happy?

**Comment by sebweb3r** [comment]
Yes LGTM!

**Pull request merged**